lib/suse/connect/api.rb

```diff
@@ -7,30 +7,34 @@ module Connect
     class Api
 
       class InvalidCredentialsError < StandardError; end
+      SYSTEM_UUID_LOCATION = 'config/system_uuid'.freeze
+      ROOT_PATH = Pathname.new(File.dirname(__FILE__)).join('../', '../', '../').freeze
+      CONNECT_API_URL = 'https://scc.suse.com/connect'.freeze
+      UUID_FILE_LOCATION = ROOT_PATH.join(SYSTEM_UUID_LOCATION).freeze
```

Comment:
`UUID_FILE_LOCATION = File.expand_path('../../../config/system_uuid', __dir__)`.
`SYSTEM_UUID_LOCATION` and `ROOT_PATH` aren't used anywhere and shouldn't be needed.